manifest/build.go

```diff
@@ -1,18 +1,37 @@
 package manifest
 
-import "fmt"
+import (
+	"fmt"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/convox/rack/cmd/convox/stdcli"
+)
 
 func (m *Manifest) Build(dir string, s Stream, noCache bool) error {
 	pulls := map[string]string{}
 	builds := []Service{}
 
+	abs, err := filepath.Abs(dir)
+	if err != nil {
+		return err
+	}
+
+	appName := stdcli.ReadSetting("app")
```

Comment:
This package should be independent of the CLI. App name should be maybe passed in to the function.